package.json

```diff
@@ -26,18 +26,19 @@
   "scripts": {
     "start": "run-s -n init:generated fetch start-only",
     "start-only": "node ./antwar.bootstrap.js develop",
-    "build": "npm run init:generated && npm run fetch && rm -rf build/ && node ./antwar.bootstrap.js build && npm run sitemap && echo webpack.js.org > build/CNAME",
+    "build": "npm run init:generated && npm run fetch && rm -rf build/ && node ./antwar.bootstrap.js build && npm run sitemap && npm run chapterize && echo webpack.js.org > build/CNAME",
```

Comment:
For now i suggest to remove this change from here and add it after rebuild is merged for less conflicts

Reply:
Done.